docs/user_guide/get_started.rst

```diff
@@ -16,14 +16,16 @@ You can download the complete code discussed in this section :download:`here <..
 your virtualenv under ``share/radical.entk/user_guide/scripts``.
 
 
-Use pre-configured MongoDB
-==========================
+.. comments
 
-For the purposes of this user guide, we have a MongoDB setup to use. Please run the following command to use it:
+        Use pre-configured MongoDB
```

Comment:
I would eliminate the whole 'section' `Use pre-configured MongoDB` as we do not support mlab anymore.